extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -551,15 +551,40 @@ describe('Project: database references', function (): void {
 		should(ref).equal(undefined, 'msdb reference should be deleted');
 	});
 
-	it('Should add system database reference correctly', async function (): Promise<void> {
+	it('Should add system database artifact reference correctly', async function (): Promise<void> {
 		let project = await testUtils.createTestSqlProject(this.test);
 
-		const msdbRefSettings: ISystemDatabaseReferenceSettings = { databaseVariableLiteralValue: systemDatabaseToString(SystemDatabase.MSDB), systemDb: SystemDatabase.MSDB, suppressMissingDependenciesErrors: true };
+		const msdbRefSettings: ISystemDatabaseReferenceSettings = {
+			databaseVariableLiteralValue: systemDatabaseToString(SystemDatabase.MSDB),
+			systemDb: SystemDatabase.MSDB,
+			suppressMissingDependenciesErrors: true,
+			systemDbReferenceStyle: SystemDbReferenceStyle.ArtifactReference
+		};
+		await project.addSystemDatabaseReference(msdbRefSettings);
+
+		(project.databaseReferences.length).should.equal(1, 'There should be one database reference after adding a reference to msdb');
+		(project.databaseReferences[0].referenceName).should.equal(msdbRefSettings.databaseVariableLiteralValue, 'databaseName');
+		(project.databaseReferences[0].suppressMissingDependenciesErrors).should.equal(msdbRefSettings.suppressMissingDependenciesErrors, 'suppressMissingDependenciesErrors');
+		const projFileText = (await fs.readFile(project.projectFilePath)).toString();
+		(projFileText).should.containEql('<ArtifactReference Include="$(SystemDacpacsLocation)');
```

Comment:
Minor, but I've generally been removing these from the ADS tests since DacFx.Projects tests does the validation for the sqlproj file.  Keeps the ADS tests to only checking the ADS-STS border.

Reply:
Yeah I didn't think it was worth it to add reference type info if it's only for tests. I added this test just to make sure that everything is getting passed down all the way correctly and the correct reference type is used.